linux/jdk/suse/build.gradle

```diff
@@ -122,7 +145,11 @@ task checkJdkSuse(type: Test) {
 
 	def containerRegistry = getContainerRegistry()
 
-	environment "PACKAGE", "$product-$productVersion-jdk"
+	if (product == "microsoft") {
```

Comment:
Hmm, I'm in two minds about this.  I wonder if we need a separate $productDisplayName or something like that. I'm thinking a future refactoring where vendor specifics don't muddy this script.

Reply:
@karianna I agree that this is hacky, but I would also like to point out that this is the currently merged solution for `linux/deb` and `linux/redhat`

Reply:
@jmjaffe37 how much time do you feel it would take to fix this both here and in the other folders? That could be a separate PR though! Would be a nice touch.

Reply:
@d3r3kk if the solution is creating a separate $productDisplayName input, this would be a trivial change that I could finish in a day or two

Reply:
The solution is fully up to you. Add an issue for you to follow up with and see what you can do.